dev/update_mlflow_versions.py

```diff
@@ -56,8 +58,8 @@ def update_versions(new_version: str, add_dev_suffix: bool) -> None:
     # R
     replace_occurrences(
         files=[Path("mlflow", "R", "mlflow", "DESCRIPTION")],
-        pattern=current_version,
-        repl=new_version,
+        pattern=f"Version: {current_version}",
+        repl=f"Version: {new_version}",
```

Comment:
Adjusted the pattern & replacement since there are some 2.0.0 version specifiers in the R DESCRIPTION file for dependencies such as `tibble` that accidentally match this pattern